src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,16 +1,157 @@
-import { Component } from 'react';
+import React, { useState } from 'react';
+import './NewMovie.scss';
 
-type Props = {};
-type State = {};
+type Props = {
+  addMovie: (movie: Movie) => void;
+};
 
-export class NewMovie extends Component<Props, State> {
-  state: State = {};
+export const NewMovie: React.FC<Props> = ({ addMovie }) => {
+  const [title, setTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
```

Comment:
use:  `is, check, should,` etc. for boolean variables
```suggestion
  const [isTitleError, setTitleError] = useState(false);
```